Week3/homework/1-step3.js

```diff
@@ -2,6 +2,8 @@
 
 function foo(func) {
   // What to do here? 
+  //As the argument is a function so I will call the argument itself.
+  return func();
 }
 
 function bar() {
```

Comment:
Excellent.  Returning the result of calling func is very important, as the foo function does not (and should not) know what func does.